### DISCUSSION TITLE
编写A.I.来模拟一个玩家

### DISCUSSION THREAD

**Discussion by lowpair** [kbengine/kbengine#1504] [Q&A]
文档中ppt里面bot脚本中介绍，可以编写A.I.来模拟一个玩家；

NPC类里面没有详细的例子；
我看PlayerAvatar就是自动的随机操作，是否基于此扩展就可以？
那NPC类是做什么用呢？

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677。
